build/scraper.mjs

```diff
@@ -227,8 +229,7 @@ class Scraper {
 
   /**
    * Formatted date string. Format: "YYYY MM DD"
-   * @typedef {string} OggDateStamp
-   */
+   *    */
```

Comment:
wth is vs code doing. please stop.

Reply:
it's actually off. Looks like this was after I ran lint:fix

Reply:
right, but lint doesn't normally fix what you've already got in the comment... i'd actually rather we not get rid of our typedef for OggDateStamp...